tests/messages/test_pofile.py

```diff
@@ -1014,3 +1014,10 @@ def test_issue_1087():
 "Language: \n"
 ''')
     assert pofile.read_po(buf).locale is None
+
+def test_issue_1134():
+    buf = StringIO('''msgid "this is an invalid po file"
+msgstr "hello world"
+msgid "msgid without str"''')
```

Comment:
It's IMO easier to understand what a test does when it's as short as possible:

```suggestion 
    buf = StringIO('''
msgid "foo"
"''')
```